srobo-config/local.conf

```diff
@@ -287,6 +287,10 @@ hostname:pn-base-files = "robot"
 
 DL_DIR ?= "${BSPDIR}/downloads"
 
-SROBO_VERSION = "2023.0.0"
+# Expand the free space in the image to 2GB
+IMAGE_ROOTFS_EXTRA_SPACE = "2097152"
```

Comment:
I think we're using the magic systemd.growfs stuff for this, it should expand to the whole card on first boot iirc

Reply:
This is not currently on in the image, when manualy setting growfs on the partition it fails, so that is not the whole story.
2GB is enough to install a lot of pacakges and still fits on an 8GB SD card. The final image is 426MB when compressed so I dont see that as an issue.